draft-ietf-quic-recovery.md

```diff
@@ -292,13 +298,12 @@ conditions:
 * If there are more than one unacknowledged packets, PTO SHOULD be scheduled for
   max(2*SRTT, 10ms).
 
-* If RTO is earlier, schedule a TLP alarm in its place. That is, PTO SHOULD be
-  scheduled for min(RTO, PTO).
+* If RTO (RTO, {{rto}}) is earlier, schedule a TLP alarm in its place. That is,
```

Comment:
Why the "RTO" inside the parenthesis? Did you mean to say "If RTO {{rto}} is earlier"?